osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSliderBall.cs

```diff
@@ -186,17 +186,22 @@ private bool isValidTrackingAction(OsuAction action)
 
         private Vector2? lastPosition;
 
+        private bool rewinding;
+
         public void UpdateProgress(double completionProgress)
         {
             Position = drawableSlider.HitObject.CurvePositionAt(completionProgress);
 
             var diff = lastPosition.HasValue ? lastPosition.Value - Position : Position - drawableSlider.HitObject.CurvePositionAt(completionProgress + 0.01f);
 
+            if (Clock.ElapsedFrameTime != 0)
```

Comment:
This check is redundant, right? Or perhaps it was intended to handle the case where it is 0?

Reply:
I'm not 100%, but I added it as safety to make sure the direction doesn't change when paused.

Reply:
:shrug: